docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md

```diff
@@ -6,7 +6,7 @@
 EA App currently displays a blank screen when using NorthstarProton, however Northstar will still launch assuming you have logged in to EA App at least once.
 {% endhint %}
 
-> **Check your GLIBC version.** NorthstarProton currently only supports version 2.33 and higher. Verify your installed version with `ldd --version`. If the installed GLIBC is older, use the [legacy guide](playing-on-linux-legacy-guide.md). **Ubuntu 20.04 LTS**, **Debian 11**, and **Void Linux** are known to have outdated GLIBC packages. This check does not need to be completed on Steam Deck or Steam OS.
+> **Check your GLIBC version.** NorthstarProton currently only supports version 2.33 and higher. Verify your installed version with `ldd --version`. **Ubuntu 20.04 LTS**, **Debian 11**, and **Void Linux** are known to have outdated GLIBC packages, meaning you cannot use NorthstarProton on these distros and will have to use Lutris. This check does not need to be completed on Steam Deck or Steam OS.
```

Comment:
Proton and Lutris are kinda different things. I'd change it to something like

> "... meaning you will have to experiment with finding a Proton version that works for you. It is therefore strongly recommended to use an up-to-date Linux distro to be able to make use of NorthstarProton."

Reply:
making the change as suggested, though leaving the Lutris part in since it *is* a working alternative to Proton.